File: solutions/codechef/COVID19/COVID19_3.py
# DATE: 07/01/2023, 07:52:04
# PROBLEM NAME: Coronavirus Spread
# PROBLEM URL: https://www.codechef.com/problems/COVID19
# PROBLEM DIFFICULTY RATTING: 1219
# STATUS: accepted
# TIME: 0.04
# MEMORY: 9.5M

for _ in range(int(input())):
    n = int(input())
    x = list(map(int, input().split()))
    lst = []
    last = 0

    for i in range(n - 1):
        if x[i + 1] - x[i] > 2:
            lst.append(x[last: i + 1])
            last = i + 1

    lst.append(x[last:])

    print(len(min(lst, key=len)), len(max(lst, key=len)))

File: solutions/codechef/COVID_19/COVID_19_3.c
// DATE: 03/11/2022, 06:26:24
// PROBLEM NAME: Covid and Theatre Tickets
// PROBLEM URL: https://www.codechef.com/problems/COVID_19
// PROBLEM DIFFICULTY RATTING: 1077
// STATUS: accepted
// TIME: 0.00
// MEMORY: 5.3M

#include <stdio.h>

int main(void) {
    int t, n, m;
    scanf("%d", &t);
    
    while(t--){
        scanf("%d%d", &n, &m);
        printf("%d\n", (n/2 + n%2)*(m%2 + m/2));
    }
    
	return 0;
}



